### PR TITLE
DPC-1439 feature: DPC v2: POST /api/v2/implementer (proxy to Attribution)

### PR DESCRIPTION
**Pull request by charmcitygavin**
### Fixes [DPC-1439](https://jira.cms.gov/browse/DPC-1439)

Proxy `POST` requests sent to `/api/v2/implementer` to the Attribution service to create an implementer and return that new implementer

### Proposed Changes

* Add an implementer controller to handle the proxy request
* Add a route for the implementer
* Add an Implementer `ResourceType` to the attribution client
* Write tests

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

Screen Shot 2021-06-28 at 11 22 28 AM

### CODE REVIEW

File: dpc-go/dpc-api/src/v2/implementer.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+// NewImplementerController function that creates an implementer and returns its reference
+func NewImplementerController(ac client.Client) *ImplementerController {
+	return &ImplementerController{
+		ac,
+	}
+}
+
+// Create function is not currently used for ImplementerController
```
Comment (MrMorie) [inline dpc-go/dpc-api/src/v2/implementer.go:24]:
```suggestion
// Create function is used for creating a new implementor by proxying request to dpc attribution
```

**Review by MrMorie — DISMISSED** [review]
LGTM

File: dpc-go/dpc-api/src/v2/implementer.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+// NewImplementerController function that creates an implementer and returns its reference
+func NewImplementerController(ac client.Client) *ImplementerController {
+	return &ImplementerController{
+		ac,
+	}
+}
+
+// Create function is used for creating a new implementor by proxying request to dpc attribution
```
Comment (dhgreene) [inline dpc-go/dpc-api/src/v2/implementer.go:24]:
```suggestion
// Create function is used for creating a new implementer by proxying request to dpc attribution
```

Comment (charmcitygavin) [reply dpc-go/dpc-api/src/v2/implementer.go]: Great catch. Thank you!

**Review by dhgreene — APPROVED** [review]
👍

**Pull request merged**